CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## NEXT - TBD
 ### Fixed
+- FSDP: fix extra process groups
```

Comment:
Should we add the PR link here and also specify why we fixed it?

Reply:
Thank you for the review! I think it might be a bit too much manual work to maintain the PR links. The info is already there in the git, replicating it here by hand seems to consume too much time that could be use elsewhere.

Good idea on specify why. I will add more info on this line.

Reply:
I think it is very hard to find the exact PR if we need to post release. Can we at least add the PR number? I think that is sufficient information for us to track down issues.

Reply:
Sure. Sounds like a good tradeoff for now.